transcom/mymove
# MB-4830 Bug initial authorized weight should be same as weight allowance

## Description

> According to LTC Ryan, weight entitlement/allowance should not include pro gear. We still want to display the amounts for pro gear and spouse pro gear, like we already to in the TOO interface. However, only rank and dependents should be used to determine the initial weight allowance and authorized allowance. 
> 
> ## Acceptance Criteria
> 
> Write a test to verify the existing broken behavior
> 
> Make the test pass by modifying the `GetEntitlement` function to only take the rank and dependents and return the appropriate number
> 
> Modify all the places where we call `GetEntitlement`. 

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

## To test
1. Log in to office app as TOO role, locally
2. Click on any test move (non of the moves should have a set authorized weight)
3. Scroll down to `Allowances`
![image](https://user-images.githubusercontent.com/1522549/100281512-e906e680-2f1e-11eb-879d-4ce9c102ece0.png)
4. Weight Allowance and Authorized Weight should be the same

Notes:
- To test for dependents_authorized is false, you can quickly change the value in the database to see the changes in the `entitlements` table

## Code Review Verification Steps


 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4830) for this change
